src/__tests__/act.test.tsx

```diff
@@ -43,7 +43,7 @@ test('fireEvent should trigger useState', () => {
 });
 
 test('should act even if there is no act in react-test-renderer', () => {
```

Comment:
I'd assume this test is not needed anymore.

Reply:
Good catch, I've removed that tests. Funny thing is that it did not fail despite no longer supporting missing `act`.